worker/peergrouper/initiate.go

```diff
@@ -6,19 +6,20 @@ package peergrouper
 import (
 	"time"
 
+	"github.com/juju/clock"
 	"github.com/juju/errors"
 	"github.com/juju/mgo/v2"
 	"github.com/juju/replicaset/v2"
-	"github.com/juju/utils/v3"
+	"github.com/juju/retry"
 
 	"github.com/juju/juju/agent"
 	"github.com/juju/juju/mongo"
 )
 
-// TODO(katco): 2016-08-09: lp:1611427
-var initiateAttemptStrategy = utils.AttemptStrategy{
-	Total: 60 * time.Second,
-	Delay: 5 * time.Second,
+var initiateAttemptStrategy = retry.CallArgs{
+	Clock:       clock.WallClock,
```

Comment:
Use the clock passed into the worker in the config.

Reply:
I don't think I have access to a clock like this? Since this isn't a method of `pgWorker`, and isn't passed in a worker as a param. Or is there something I'm missing here?